### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by Satansstuff** [vizs/manjarno#9]
Many of these things you bring up are indeed problems, but since you do not give a "Do this instead of that" 
It just makes it look like you are really angry at Manjaro for some reason, and just want to find flaws instead of making things better.

Also if you have a package X that rely on package Y, then it's up to package X to specify what version of package Y it depends on (not that every package developer does this and this is why the AUR is pretty trash and should be avoided unless absolutely necessary). 

"Their own updater had a security vulnerability which wasn't fixed until recently" 
So you mean that a feature in an operating system widely used had a security vulnerability? Wow, that is totally not something that happens all the time. What would be disconcerting here is if the developers took a long time to fix the issue, since it's a big part of the distro.

"Manjaro let their SSL certificates expire not once but twice"
Yes that is indeed kind of a fail, but not like it's the end of the world. Hopefully they learned their lesson and will make sure it won't happen a third time.

Just to clarify, I do not really like Manjaro. I feel that it goes against the Arch ideal of being extremely lightweight and the fact that you don't get the ability to build the system from scratch. However, I do think that it can be useful for a beginner to dual boot / run in a VM to let them acclimate to an arch type environment since they are probably changing from some "simpler" type of dist.

Peace

**Comment by shantaram3013** [comment]
I agree, I think at the very least an alternative should be suggested (such as EndeavourOS).

**Comment by erikKeresztes** [comment]
just use anarchy, it's vanilla arch but with an easy CLI installer

**Comment by Atakku** [comment]
> Many of these things you bring up are indeed problems, but since you do not give a "Do this instead of that"
Well "**Install arch instead of Manjaro**" :^)

**Comment by binex-dsk** [comment]
@erikKeresztes you might as well use Arch at that point because at least THEN you get to know how to use the command line which is... the whole point of Arch

**Comment by Atakku** [comment]
> @erikKeresztes you might as well use Arch at that point because at least THEN you get to know how to use the command line which is... the whole point of Arch

nah the whole point of Arch is being a minimal distro where you pick everything you want yourself, but can't deny, arch taught me a lot of things over the past 3 years :P

**Comment by binex-dsk** [comment]
> > @erikKeresztes you might as well use Arch at that point because at least THEN you get to know how to use the command line which is... the whole point of Arch
> 
> 
> 
> nah the whole point of Arch is being a minimal distro where you pick everything you want yourself, but can't deny, arch taught me a lot of things over the past 3 years :P

ya thats what i meant. You can't really get the "minimal" distro without familiarity of the command line, because about 95% of the things you can do on Linux are command line only. also, when using Arch you get to fully utilize the Arch Wiki which is pretty fuckin epic ngl

**Comment by erikKeresztes** [comment]
> @erikKeresztes you might as well use Arch at that point because at least THEN you get to know how to use the command line which is... the whole point of Arch

i agree, but it's just a waste of time once you already know the commands

**Comment by binex-dsk** [comment]
> > @erikKeresztes you might as well use Arch at that point because at least THEN you get to know how to use the command line which is... the whole point of Arch
> 
> 
> 
> i agree, but it's just a waste of time once you already know the commands

for me ive basically memorized the whole thing, only takes a few
minutes unless my internet is doing a schizo moment

but yeah sometimes it does feel like a waste of time, that is until I remember that Windows 8.1 takes 20 minutes to install on a modern Ryzen and NVMe SSD and i no longer hold that opinion

**Comment by vizs** [comment]
I did plan on rewriting this sometime during the pandemic but I didn't have the motivation so I delayed it (indefinitely). I contemplated on archiving manjarno and link to other manjarno-esque write-ups.

However, I did get the motivation recently but my end sem starts tomorrow (in my time) so this will have to wait until after the New Years. In the meantime, I'd appreciate if you could point how to improve the writing.

**Comment by Satansstuff** [comment]
> I did plan on rewriting this sometime during the pandemic but I didn't have the motivation so I delayed it (indefinitely). I contemplated on archiving manjarno and link to other manjarno-esque write-ups.
> 
> However, I did get the motivation recently but my end sem starts tomorrow (in my time) so this will have to wait until after the New Years. In the meantime, I'd appreciate if you could point how to improve the writing.

Well, I suggest being more objective.

    

- Bring up why people feel inclined to use Manjaro in the first case.
- Objectively talk about why the use of Manjaro does the user no good in the long term.
- Bring up mistakes and errors they have repeatedly done in the past, but make sure to write that these problems have been fixed (if they have been naturally).
- Bring up flaws with the Dist as a whole.
- Bring up flaws with the distribution of packages in manjaro compared to arch itself.
- Motivate why a new user to the Arch-typed distribution should go with Arch instead of Manjaro (or other Manjaro type derivations).

Just some stuff of the top of my head, since you asked; but I'm sure if you feel inclined you can find better points.
Just keep emotions out of the post and maybe finally we can get people to stop using manjaro as a stopgap and go with arch from the very beginning. 
Good luck with your end-sem and have a happy new-year.
